spicy/toolchain/src/compiler/codegen/parsers/types.cc

```diff
@@ -128,6 +128,8 @@ struct Visitor : public hilti::visitor::PreOrder<Expression, Visitor> {
         performUnpack(target, t, t.width() / 8, {state().cur, fieldByteOrder(), *bitorder}, t.meta(), is_try);
 
         if ( pb->options().debug ) {
+            auto have_value = builder()->addIf(builder::hasMember(target, "__value__"));
+            pushBuilder(have_value);
```

Comment:
Nit: this would nicely fit use of `pushBuilder` with a lambda so we wouldn't have too use `popBuilder()` below.